src/main/java/com/urswolfer/gerrit/client/rest/http/changes/RevisionApiRestClient.java

```diff
@@ -19,11 +19,9 @@
 import com.google.common.base.Strings;
 import com.google.gerrit.extensions.api.changes.*;
 import com.google.gerrit.extensions.client.SubmitType;
-import com.google.gerrit.extensions.common.CommentInfo;
-import com.google.gerrit.extensions.common.CommitInfo;
-import com.google.gerrit.extensions.common.FileInfo;
-import com.google.gerrit.extensions.common.TestSubmitRuleInput;
+import com.google.gerrit.extensions.common.*;
 import com.google.gerrit.extensions.restapi.BinaryResult;
+import com.google.gerrit.extensions.restapi.NotImplementedException;
```

Comment:
unneeded?

Reply:
Thank you, I missed that. Now, it's fixed